lib/signaling/v2/peerconnectionmanager.js

```diff
@@ -126,11 +144,15 @@ class PeerConnectionManager extends QueueingEventEmitter {
           peerConnection.removeListener('stateChanged', stateChanged);
           self._peerConnections.delete(peerConnection.id);
           self._closedPeerConnectionIds.add(peerConnection.id);
+          updateIceConnectionState(self);
         }
       });
+      peerConnection.on('iceConnectionStateChanged', () => updateIceConnectionState(this));
```

Comment:
@markandrus I think we should make `updateIceConnectionState()` a member of `PeerConnectionManager`. WDYT?

Reply:
I don't really like to add members to the prototype if it can be avoided. Perhaps TypeScript will force this, though. But I'd rather defer doing this for now, since this is a pattern we have elsewhere: functions for mutating private members scoped to the module instead of the prototype.

Reply:
Should we be removing this listener at some point? Destruction? If so, we'll need a bound version of this attached to the instance I believe.

Reply:
We don't remove the others, so I don't think we need to remove this one. Or rather, if we decide we do need to remove these, let's tackle in a separate PR.